src/wallet/test/coinselector_tests.cpp

```diff
@@ -136,6 +136,14 @@ inline std::vector<OutputGroup>& GroupCoins(const std::vector<COutput>& coins)
     return static_groups;
 }
 
+inline std::vector<OutputGroup>& KnapsackGroupOutputs(const CoinEligibilityFilter& filter)
+{
+    static std::vector<OutputGroup> static_groups;
+    static_groups.clear();
+    static_groups = testWallet.GroupOutputs(vCoins, coin_selection_params, filter, false);
```

Comment:
How come this uses "GroupOutputs" when BnB uses "GroupCoins"? What's the difference?

Reply:
I've added an explanation in the commit message. The reason is because the `KnapsackSolver` tests involve different eligibility filters which `GroupCoins` does not.